### PR TITLE
📃 Add step to release doc

### PR DESCRIPTION
**Pull request by sofokli1**
Add step to create a test issue for the release using the `Manual Test Template`

### CODE REVIEW

**Comment by sofokli1** [comment]
I have 2 question about this:
1. Is the position of the new step appropriate? Should it be after the tag creation?
2. Should the release manger also add extra tests to the list for the specific changes of the release?

**Comment by amulware** [comment]
2. yes, definitely. There should probably be even a section in the issue template for this.

**Comment by amulware** [comment]
Please also see about updating https://github.com/toggl/mobile-docs/blob/develop/release-flow.md if needed.

**Comment by amulware** [comment]
Also maybe worth mentioning that the issue and release PR should be linked. (I think if the PR is mentioned in the issue, we can see the checkbox progress in that reference? That might be convenient.)

File: superflow.md [hunk 1]
```diff
@@ -108,6 +108,7 @@ From the above it follows that these are the steps to release a new version of t
 - Create a release branch from develop. This contains all the changes wanted in this new version.
```
Comment (sofokli1) [inline superflow.md]: @amulware I would propose to completely remove the steps from here and instead link straight to the `Toggl Mobile App Release Flow`. There would be only one source of truth. What do you think?

Comment (amulware) [reply superflow.md]: I'm fine with that, if you and others think that the new document is clear. (And if not, we can fix that of course)

**Pull request merged**